server/src/database/seeds/AdminClientSeeder.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Gernzy\Server\Database\Seeds;
+
+use Gernzy\Server\Models\User;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+
+// use Illuminate\Support\Facades\DB;
+
+class AdminClientSeeder extends Seeder
+{
+    protected $foo = [];
```

Comment:
@AlexeiaW this may have come in from an example, but we can remove this var.

Reply:
thanks, removed that